### PR TITLE
Add Github Issue Template

### PR DESCRIPTION
**Pull request by kieraneglin**
Does what it says on the tin! If I missed anything, please let me know.  I tried to follow the [issue submission guidelines](https://github.com/diesel-rs/diesel/blob/f7bb506ffc62ec72aafe60d7909d3614e3ff5376/CONTRIBUTING.md#submitting-bug-reports) as closely as possible.

For #969

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Before submitting an issue, please check over the [issue tracker](https://github.com/diesel-rs/diesel/issues) to see if your problem has been discussed before.
```
Comment (killercup) [inline .github/ISSUE_TEMPLATE.md]: This is good, but I fear the wall of text my scare people who do not want to report a bug. Maybe we can add something like this at the beginning?

> If you want to report a bug, we added some points below you can fill out. If you want to request a feature, feel free to remove all the irrelevant text. In addition to this issue tracker, you can also talk to Diesel maintainers and users on [Gitter](https://gitter.im/diesel-rs/diesel).

**Comment by kieraneglin** [comment]
I've updated the template per your suggestions!  I also remove the `*Description*` lines.  They didn't provide anything meaningful, so I added spaces instead.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+-->
+
+## Setup
+
+### Versions
+
+- **Rust:**
+- **Diesel:**
+- **Database:**
+- **Feature Crates:**
```
Comment (Eijebong) [inline .github/ISSUE_TEMPLATE.md]: What does that mean ?

Comment (kieraneglin) [inline .github/ISSUE_TEMPLATE.md]: If you're using Chrono as a diesel feature, what version are you using?

I thought it'd be good to add the versions of any diesel "plug-ins" they may be using.

Comment (Eijebong) [inline .github/ISSUE_TEMPLATE.md]: Well if you're using the chrono feature in diesel, the chrono version it's using is using is set by diesel.

Comment (kieraneglin) [inline .github/ISSUE_TEMPLATE.md]: Ah!  Good point!  My internal Rust novice is showing.  Will fix

**Comment by killercup** [comment]
Good changes. Let's merge this, then!

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+### What is the actual output?
+
+
+### Are you seeing any additional errors?
+
+
+### Steps to reproduce
+
+<!--
+Please include as much as your codebase as needed to reproduce the error.  If the relevant files are large, please consider linking to a public repository or a [Gist](https://gist.github.com/).
```
Comment (PeterW-LWL) [inline .github/ISSUE_TEMPLATE.md]: Please include as much **of** your codebase as needed...

Comment (killercup) [inline .github/ISSUE_TEMPLATE.md]: Good catch! Fixed in 1ad6ad8

**Pull request merged**